File: src/ERP/acme.atena.domain/DTO/Account/Receita.cs
using acme.atena.domain.DTO.Util;
using System;
using System.Collections.Generic;
using System.Text;

namespace acme.atena.domain.DTO.Account
{
    public class Receita : AbstractEntity
    {
        public Receita()
        {
            FluxoDeCaixasReceitas = new HashSet<FluxoDeCaixaReceita>();
        }

        public string Nome { get; set; }
        public string Descricao { get; set; }
        public decimal Valor { get; set; }
        public bool ReceitaFixa { get; set; }

        public Guid CompetenciaId { get; set; }

        public virtual Competencia Competencia { get; set; }

        public virtual ICollection<FluxoDeCaixaReceita> FluxoDeCaixasReceitas { get; set; }

    }
}

File: site/coreui-free-angular-admin-template-master/src/app/view-model/response/account/receita.ts
import { FluxoDeCaixaReceita } from './fluxo-de-caixa-receita';
import { AbstractEntityCompetencia } from "../abstract-entity-competencia";
import { Competencia } from "../util/competencia";

export class Receita extends AbstractEntityCompetencia{
  public nome:string;
  public descricao:string;
  public valor: number ;
  public receitaFixa: number;

  public competenciaId: string;

  public competencia: Competencia;

  public fluxoDeCaixasReceitas: FluxoDeCaixaReceita[];
}
